.github/workflows/tests.yaml

```diff
@@ -189,7 +189,7 @@ jobs:
                   ./scripts/run_in_build_env.sh \
                   "./scripts/tests/run_test_suite.py \
                      --chip-tool ./out/darwin-x64-chip-tool${CHIP_TOOL_VARIANT}-${BUILD_VARIANT}/chip-tool \
-                     --target-skip-glob '{TestGroupMessaging}' \
+                     --target-skip-glob '{TestGroupMessaging,TestArmFailSafe}' \
```

Comment:
Why is this being skipped on Mac?

Reply:
Hit some strange errors, sync to ToT and hope it can pass now

Reply:
CommissionerCommands is not available in Darwin, all yaml test cases use other APIs of this class such as PairWithQRCode are also not enabled on Darwin

Reply:
> CommissionerCommands is not available in Darwin

It's certainly available in Darwin chip-tool.

> all yaml test cases use other APIs of this class such as PairWithQRCode are also not enabled on Darwin

I think you are mixing up the "Darwin" test job and the "Tests" test job running on Darwin.  The former doesn't do CommissionerCommands.  The latter should do them fine, and runs `TestMultiAdmin` in our CI all the time.

Reply:
I am confused with "Builds / Build on Darwin" and "Darwin / Build Darwin", which one is "Darwin" test job?

After I enable  this test on Darwin, it failed in "Darwin / Build Darwin" 

```
/Users/runner/work/connectedhomeip/connectedhomeip/src/darwin/Framework/CHIPTests/CHIPClustersTests.m:50570:5: error: implicit declaration of function 'PairWithTimeout' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
    PairWithTimeout(expectation, queue, 51966, @"MT:-24J0AFN00KA0648G00", 0);
    ^
```